indexd/index/blueprint.py

```diff
@@ -374,6 +375,74 @@ def add_index_record_version(record):
 
     return flask.jsonify(ret), 200
 
+@blueprint.route("/index/<path:record>/aliases", methods=["GET"])
+def get_aliases(record):
+    """
+    Get all aliases associated with this DID / GUID
+    """
+    try:
+        aliases = blueprint.index_driver.get_aliases_for_did(record)
+        aliases_payload = {
+            "aliases": [{"value": alias} for alias in aliases]
+        }
+        return flask.jsonify(aliases_payload), 200
+    except NoRecordFound as err:
+        raise err
```

Comment:
we could add some logs here with more details including the `record` that got passed in (same for errors below)

Reply:
can use our `cdislogging` library. something like: 

```
from cdislogging import get_logger

from gen3authz.client.arborist.client import ArboristClient
logger = get_logger(__name__, log_level="info")  # TODO that info level should probably be set based on some configuration to "debug" but unsure if indexd has such a thing right now
```

and then in the funcitons:
```
def do_something(foobar):
  logger.info("info!!")
  logger.warning("warning!!")
  logger.error(f"very bad things have occurred. error: {err}")
```